src/services/dayz.js

```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+
+class Dayz {
+    axiosInstance = axios.create({
```

Comment:
good work using axios instances instead of using this base url everywhere. just a tip: add a constructor in this class and move this instance inside it